sorc/ncep_post.fd/MDL2P.f

```diff
@@ -57,8 +58,9 @@ SUBROUTINE MDL2P(iostatusD3D)
       use ctlblk_mod, only: MODELNAME, LP1, ME, JSTA, JEND, LM, SPVAL, SPL,    &
                             ALSL, JEND_M, SMFLAG, GRIB, CFLD, FLD_INFO, DATAPD,&
                             TD3D, IFHR, IFMIN, IM, JM, NBIN_DU, JSTA_2L,       &
-                            JEND_2U, LSM, d3d_on, gocart_on, ioform, NBIN_SM,  &
-                            imp_physics, ISTA, IEND, ISTA_M, IEND_M, ISTA_2L, IEND_2U
+                            JEND_2U, LSM, d3d_on, ioform, NBIN_SM,  &
```

Comment:
@zhanglikate Do you think "DUSTSL" can be removed from following statements in this routine? 
```
!     REAL D3DSL(IM,JM,27),DUSTSL(IM,JM,NBIN_DU)
 REAL, allocatable  ::  D3DSL(:,:,:), DUSTSL(:,:,:), SMOKESL(:,:,:)
if(allocated(dustsl))  deallocate(dustsl)
```

Reply:
@WenMeng-NOAA  Updated and pushed. Thanks.